ess/src/API/EMBC.ESS/Managers/Events/EventsManager.Supports.cs

```diff
@@ -101,6 +104,8 @@ public async System.Threading.Tasks.Task Handle(ProcessSelfServeSupportsCommand
             PrintReferrals = false
         });
 
+        bool emailSent = await SendEmailConfirmation(cmd.ETransferDetails, file.PrimaryRegistrantId, file.PrimaryRegistrantUserId, supports);
```

Comment:
The email should be the last thing that goes out. Yes, we need this to be refactored.

Reply:
do you mean SendEmailConfirmation should be just moved to be the last call inside the function?